Automattic/wp-calypso
# [WooCommerce NUX] Fix incorrect redirection for existing user

Related to https://github.com/Automattic/wp-calypso/issues/86884

When a user tries to sign up using Google or Apple, and their email matches an existing account that hasn't been connected yet, the user will be redirected to Woo's login screen and it will display a warning notice.

![Screenshot 2024-01-31 at 18 50 46](https://github.com/Automattic/wp-calypso/assets/4344253/68ad0afc-17c7-4909-a045-ff44e256de9b)

Figma: 4ixWMlzrxllx93tSFsCW6k-fi-5037_90936


## Proposed Changes

* Fix incorrect link when redirecting to the login page.
* Add woo version existing user warning notice to the login screen.

## Testing Instructions



1. Set up local Woo Start environment https://github.com/woocommerce/woocommerce-start-dev-env/tree/trunk to test social login.
2. Create a WP.com account with Gmail or [disconnect Google from your existing WP.com account]( https://wordpress.com/me/security/social-login) Please use non-a8c account.
3. Log out WP.com
4. Go to https://woo.com/start/
5. Continue to the account creation step
6. Sign up via Google
7. Observe that you're directed to the Woo's login screen
8. Observe that it shows `You already have a WordPress.com account with this email address. Add your password to log in or create a new account.` figma: 4ixWMlzrxllx93tSFsCW6k-fi-5037_90936
9. Confirm "Create a new account" link is valid and notice is dismissible.

## Pre-merge Checklist
